modules/cli/src/main/scala/scala/cli/ScalaCli.scala

```diff
@@ -201,14 +201,21 @@ object ScalaCli {
     // load java properties from JAVA_OPTS and JDK_JAVA_OPTIONS environment variables
     val javaOpts = sys.env.get("JAVA_OPTS").toSeq ++ sys.env.get("JDK_JAVA_OPTIONS").toSeq
 
-    javaOpts
-      .flatMap(_.split("\\s+"))
-      .foreach { opt =>
-        opt.stripPrefix("-D").split("=", 2) match {
-          case Array(key, value) => System.setProperty(key, value)
-          case _                 => // Ignore if not a Java property
-        }
-      }
+    val ignoredJavaOpts =
+      javaOpts
+        .flatMap(_.split("\\s+"))
+        .flatMap { opt =>
+          opt.stripPrefix("-D").split("=", 2) match {
+            case Array(key, value) =>
+              System.setProperty(key, value)
+              None
+            case ignored => Some(ignored) // non-property opts are ignored here
+          }
+        }.flatten
+    if ignoredJavaOpts.nonEmpty then
+      System.err.println(
+        s"Warning: Only java properties are supported in JAVA_OPTS and JDK_JAVA_OPTIONS environment variables. Other options are ignored: ${ignoredJavaOpts.mkString(", ")}"
```

Comment:
I'm now getting this message every time I invoke scala-cli. What would be the recommendation? I would like to keep that env var because sbt supports it (I'm using `-Xmx1536m -Xms1536m -Xss5m ...`).